compiler/onecc-docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:18.04 
+
+RUN apt update -y && \ 
+	 apt install -y \ 
+	 wget && \ 
+	 apt clean 
```

Comment:
```suggestion
RUN apt update -y && \ 
    apt install -y wget && \ 
    apt clean 
```

Reply:
please do not use TABs

Reply:
I realized that this code don`t pass nacc format
I will check well before making pull request next time